### DISCUSSION TITLE
How to create mongoose_fs.c

### DISCUSSION THREAD

**Discussion by ahmetbudak1** [cesanta/mongoose#2948] [Ask questions, get answers] [answered]
please help me.

How to create static const unsigned char v1[]
array in mongoose_fs.c?

**Comment by scaprile** [comment, marked as answer]
Please see our [documentation](https://mongoose.ws/documentation/),  and follow the guidelines in our [tutorials](https://mongoose.ws/documentation/#tutorials-and-examples).
https://mongoose.ws/documentation/tutorials/core/embedded-filesystem/
